.github/workflows/ruby.yml

```
@@ -26,7 +26,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby-version: [2.5, 2.7]
+        ruby-version: [2.7, 3.0, 3.1]
```

```suggestion
        ruby-version: ['2.7', '3.0', '3.1']
```

Otherwise it interrupts this as just 3 which pulls the latest Ruby version.

---

@archanaserver Can you tackle this?

---

Sure! I'm looking into it, now seems like there is an compatibility version issue in `mocha` and `minitest`.